extensions/blocks/story/index.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+import { Path, SVG } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import save from './save';
+import getCategoryWithFallbacks from '../../shared/get-category-with-fallbacks';
+
+/**
+ * Example Images
+ */
+import storyExample1 from './story_example-1.png';
+
+export const icon = (
+	<SVG xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+		<Path d="M0 0h24v24H0z" fill="none" />
+		<Path
+			fillRule="evenodd"
+			clipRule="evenodd"
+			fill="#2C3338"
+			d="M6 3H14V17H6L6 3ZM4 3C4 1.89543 4.89543 1 6 1H14C15.1046 1 16 1.89543 16 3V17C16 18.1046 15.1046 19 14 19H6C4.89543 19 4 18.1046 4 17V3ZM18 5C19.1046 5 20 5.89543 20 7V21C20 22.1046 19.1046 23 18 23H10C8.89543 23 8 22.1046 8 21H18V5Z"
+		/>
+	</SVG>
+);
```

Comment:
Since we're using a regular Play icon, do you think we could rely on one of the media dashicons instead? I think you could use the dashicon name when registering the block here in this file, and rely on `BlockIcon` with passing the dashicon name elsewhere.
https://developer.wordpress.org/resource/dashicons/

Reply:
I recently updated it, it's a custom icon now, are you sure you were testing the last version?

Reply:
I think I was. How custom is it? It looks like a Play icon to me, but I'm no designer :)

Reply:
It should look like this 
![image](https://user-images.githubusercontent.com/230230/88517210-cc00ec00-cfee-11ea-8b0c-32e556f73216.png)

I replaced it a couple hours before your review though so you may have started the review without [this commit](https://github.com/Automattic/jetpack/pull/16302/commits/4dc7848cb412a02ea688776ad53deca67ea0b13b).